expat/coverage.sh

```diff
@@ -102,9 +102,6 @@ _run() {
 
     ${with_unsigned_char} && BASE_FLAGS="${BASE_FLAGS} -funsigned-char"
 
-    local CFLAGS="-std=c99 ${BASE_FLAGS}"
-    local CXXFLAGS="-std=c++98 ${BASE_FLAGS}"
```

Comment:
I do not know what was the intended purpose of such local definitions; I only see that they are unused.  You should contact the original author if possible.  Maybe the script was meant to be modified by hand to use such definitions?

Right now I have not tried to do the full coverage inspection operation, because I have a bit of trouble installing and running `lcov`.

Reply:
>  You should contact the original author if possible.

That's easy, it's me :smile: 

```
# git blame coverage.sh | grep 'C\(XX\)\?FLAGS'
5f5114540 (Sebastian Pipping 2017-07-05 22:49:51 +0200 105)     local CFLAGS="-std=c99 ${BASE_FLAGS}"
a340b037e (Sebastian Pipping 2017-05-06 23:51:42 +0200 106)     local CXXFLAGS="-std=c++98 ${BASE_FLAGS}"
a340b037e (Sebastian Pipping 2017-05-06 23:51:42 +0200 113)                 CFLAGS="${BASE_FLAGS}" \
a2ae7cd30 (Sebastian Pipping 2017-05-08 20:20:50 +0200 114)                 CXXFLAGS="${BASE_FLAGS}"
```
I propose to fix like:
```
        _configure \
                CFLAGS="${CFLAGS}" \
                CXXFLAGS="${CXXFLAGS}"
```
Any objections? Would you like to drop `local`?

Reply:
No objections.